src/index.ts

```diff
@@ -2,3 +2,24 @@ export * from './components';
 export * from './hooks';
 export * from './models';
 export * from './utils';
+
+/**
+ * Content path to the source files in component library that utilizes Tailwind class names.
+ * This is intended to be used in user's custom tailwind config.
+ *
+ * @public
+ *
+ * @remarks
+ * This assumes that the node_modules folder containing the component library
+ * is in adjacent level with the user's tailwind.config.js file.
+ *
+ * @example
+ * In user's tailwind.config.js file:
+ *
+ * const \{ AnswersTailwindPath \} = require('\@yext/answers-react-components')
+ * module.exports = \{
+ *  content: [ AnswersTailwindPath ],
+ *  ...
+ * \};
+ */
+export const AnswersTailwindPath = 'node_modules/@yext/answers-react-components/lib/**/*.{js,ts,jsx,tsx}';
```

Comment:
In `lib` will there be `ts` and `tsx` files? I thought they would all be trans-piled to JS/JSX at that point. Also, I think the name `AnswersTailwindPath` is a bit confusing. It sounds like we're pointing them to our Tailwind config. We're pointing them to the package's Tailwind "content" in a sense.

Reply:
right, I forgot to update! I originally had it point to  'node_modules/@yext/answers-react-components/**/*.{js,ts,jsx,tsx}' which includes other ts/tsx files outside of lib/. During testing, test-site would get an error about recursive content paths since that glob file path includes test-site's stuff as well. I updated it to restrict to lib/, user shouldn't be importing anything outside of that anyway.
For the name, maybe "componentSourcePath" or "componentContentPath"?

Reply:
Yeah, something like `ComponentsContentPath` sounds good!